docs/Blog-Post.md

```diff
@@ -0,0 +1,578 @@
+# ---
+layout: blog
+title: Levanter —  Legible, Scalable, Reproducible Foundation Models with JAX
```

Comment:
NIT - it seems a verb is missing in the title, like "Train Legible, Scalable, Reproducible Foundation Models with JAX"

Reply:
hrm, I think as is is better. Name - Thing you get with Name is not an uncommon pattern for titles i think